src/v/schema_registry/schema_registry.h

```diff
@@ -14,7 +14,7 @@
 #include "pandaproxy/schema_registry/api.h"
 #include "pandaproxy/schema_registry/types.h"
 
-namespace wasm {
+namespace schema_registry {
```

Comment:
The double name is not my favorite.

Options:

sr::schema_registry

or

schema::registry


Maybe we name the module schema and go with the later bc it's really for consumers dealing with schemas instead of the actual schema registry implementation